CHANGELOG.rst

```diff
@@ -5,17 +5,18 @@ Change Log
    This file loosely adheres to the structure of https://keepachangelog.com/,
    but in reStructuredText instead of Markdown.
 
-2020-08-15
+2023-03-16
 **********
 
-.. I wish we used a tool to manage changelogs so that I didn't have to choose
```

Comment:
lol

Reply:
I mean, he's not wrong! I do want to look into using scriv at some point.